rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.rollover/25_max_shard_doc_condition.yml

```diff
@@ -0,0 +1,59 @@
+---
+"Rollover with max_primary_shard_docs condition":
+  - skip:
+      version: " - 8.0.99"
```

Comment:
Master is now 8.2, so maybe after you've merged in master can you change this to `8.1.99`?

Reply:
done